docs/conf.py

```diff
@@ -0,0 +1,308 @@
+# -*- coding: utf-8 -*-
+#
+# PyPOM documentation build configuration file, created by
+# sphinx-quickstart on Thu Dec 10 11:38:25 2015.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys
+import os
+import shlex
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#sys.path.insert(0, os.path.abspath('.'))
```

Comment:
Building the docs fails to perform any imports for me, but this can be fixed by adding the following line here:

```py
sys.path.insert(0, os.path.abspath('..'))
```